File: backend/tiempomaya/src/main/java/com/ts2/tiempomaya/modelo/Concepto.java
package com.ts2.tiempomaya.modelo;

import lombok.*;

import javax.persistence.*;

@Entity
@Table(name = "Concepto")
@Getter
@Setter
@ToString
@Builder
@AllArgsConstructor
@NoArgsConstructor
public class Concepto {

    @Id
    @Column(name = "palabra")
    private String palabra;
    @Column(name = "significado")
    private String significado;

}

File: frontend/tiempomaya/src/app/modelo/Concepto.ts
export class Concepto{
    idConcepto:number;
    palabra:string;
    significado:string;

    constructor(
        idConcepto:number,
        palabra:string,
        significado:string,
    ){
        this.idConcepto = idConcepto;
        this.palabra = palabra;
        this.significado = significado;

    }
}